src/main/cpp/faultinj/faultinj.cu

```diff
@@ -366,6 +366,11 @@ void readFaultInjectorConfig(void)
   //
   const std::string dynamicConfigKey = "dynamic";
 
+  // An unsigned int to seed the random number generator to deterministically
+  // recreate a fault sequence
+  //
+  const std::string seedKey = "seed";
```

Comment:
seems odd that this isn't an unsigned int.

Reply:
this is just the name of the key in the config JSON. The value is an unsigned int https://github.com/NVIDIA/spark-rapids-jni/pull/460/files#diff-8718439339d6c151ff2ae2c04e7642b1c81e6a7c3c02397d34bfb525d9cf4323R389-R390

Reply:
I get it now. I was missing the key part wasn't saying read this string as an unsigned int, but instead read the json and find a key named this string and treat the value as an unsigned int. I see. I thought it was a default value that wouldn't properly evaluate into an unsigned int to force the `value_or` or path.

Thanks for the clarification.

Reply:
oh I see this can be confusing